pilot/cmd/pilot-discovery/main.go

```diff
@@ -42,6 +44,9 @@ import (
 	"istio.io/istio/pilot/tools/version"
 )
 
+const inMemoryConfigStore = "Memory"
+const kubernetesConfigStore = "Kubernetes"
```

Comment:
are these used anywhere?

Reply:
@ijsnellf initially we declared them as constants to make the switch case below more readable, however we are in process of basing this PR off of #2003 , so most likely they will go away.